fixtures/dummy/app/views/application_view.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+module Views
```

Comment:
I created this because I had to add Translation to Articles::Form view, so I took the opportunity to create something more similar to recommended usage from documentation/generator.
But I can remove it if this isn't wanted.